app/models/time_zone_geometry.rb

```diff
@@ -73,6 +74,7 @@ def load_shapefile( shp_path, options = {} )
       pg_string = {
         dbname: ApplicationRecord.connection_db_config.configuration_hash[:database],
         host: ApplicationRecord.connection_db_config.configuration_hash[:host],
+        port: ApplicationRecord.connection_db_config.configuration_hash[:port],
```

Comment:
This will cause errors if the port is *not* specified in the `configuration_hash`, and it doesn't need to be if the default will work, so please make this work in case this value is blank (same applies to line 25).

Reply:
Thanks, addressed in [c781dbc](https://github.com/inaturalist/inaturalist/pull/3705/commits/c781dbcb221bc402ea6991e8e0e13b8d848d91b6).